spec/features/create_a_document_api_down_spec.rb

```diff
@@ -42,5 +42,6 @@ def then_i_see_the_document_exists
   def and_the_preview_creation_failed
     expect(@request).to have_been_requested
     expect(page).to have_content "Error creating preview"
+    expect(Document.last.publication_state).to eq "error_sending_to_draft"
```

Comment:
Could we check this using the page content instead - something visible to the user?

Reply:
I would like to wait with that. There's no design for things like the error states yet, and I expect it to be different from what we do at the moment.

Reply:
I feel like this test is a bit strange as a feature test anyway - should we have a controller test that just checks that controller responds correctly and appropriate states change? 

Feels quite specific for a high level feature test.